WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -107,8 +107,11 @@ private extension CartView {
 }
 
 #if DEBUG
+import class Yosemite.PointOfSaleOrderService
+import enum Networking.Credentials
```

Comment:
The View shouldn't need any reference to Networking, since this is only needed for Previews, perhaps we can make a version for Previews only, encapsulated elsewhere?

The same for other views:
- ItemCardView
- ItemGridView
- ItemListView
- ...

Reply:
PointOfSaleOrderService and Credentials are used to create CartView for the preview. That is why I put the imports inside of #if DEBUG to import it just for the previews.